### PR TITLE
Add enum extension to README

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,56 @@
...
+enum SectionType: Int {
+    case title, price, others, delete, count
+}
+
+override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return SectionType.count.rawValue
+}
+```
+
+After adding **SweetSwift** you would do something like this instead:
```
Comment (marijnschilling) [inline README.md]: I think you can better say: "you can do this instead". In reference to the prior example the code below is just what you would do instead with SweetSwift, not something like it but exactly that.

**Comment by elland** [comment]
We really need that `NSFetchedResultsController` replacement, that can give you an item for an index path, instead of an item for an index.

**Comment by 3lvis** [comment]
@Elland That has something to do with this PR? I don't follow.

File: README.md (hunk 1 above)
Comment (3lvis) [reply README.md]: ✅

**Comment by elland** [comment]
It has something to do with the use case of sections there. 👍

**Comment by 3lvis** [comment]
@Elland Maybe not since this is for static table views. Or am I wrong?

**Comment by elland** [comment]
@3lvis not sure. 😅 Maybe it doesn't apply to _static_ table views... Whatever that means outside of Storyboard...

**Comment by 3lvis** [comment]
Merging this. Let me know if I should change something else and I'll do it in other PR :)

**Pull request merged**